bigbone/src/main/kotlin/social/bigbone/api/entity/Conversation.kt

```diff
@@ -0,0 +1,33 @@
+package social.bigbone.api.entity
+
+import com.google.gson.annotations.SerializedName
+
+/**
+ * Represents a conversation with "direct message" visibility.
+ * @see <a href="https://docs.joinmastodon.org/entities/Conversation/">Mastodon API Conversation</a>
+ */
+data class Conversation(
+    /**
+     * The ID of the conversation in the database.
+     */
+    @SerializedName("id")
+    val id: String? = null,
```

Comment:
Entity IDs are not nullable in all other cases we have. Change this to `val id: String = "0"`?

Reply:
Sure, thx!